InaraResponder/InaraResponder.cs

```diff
@@ -310,6 +314,20 @@ public void Handle(Event theEvent)
             }
         }
 
+        private void handleCarrierJumpedEvent(CarrierJumpedEvent @event) 
+        {
+            Ship currentShip = ((ShipMonitor)EDDI.Instance.ObtainMonitor("Ship Monitor")).GetCurrentShip();
```

Comment:
Not entirely keen on dynamic access via name to get the current ship because it hides a dependency in the project structure. Not sure if @GioVAX is aware of such code.

Reply:
Fair, though it is structure that we have had to use at a number of locations (including other instances in the Inara Responder).

Reply:
In general I'm a big fan of explicit `using` declarations wherever available, as they clarify the dependency tree for both code and projects.

Reply:
I saw this kind of coupling in other parts of the code while working on the EddiCore project, and I have an idea to tackle this in a general way, that I am going to discuss in the team chat.

For the moment, this code is consistent with the rest of the solution, so it's fine for me.